### ISSUE TITLE
Question: can I use the ban rule, but also whitelist certain functions?

### ISSUE THREAD

**Issue by chrismbarr** [palantir/tslint#3041]
Since our application still needs to support IE11, and the `array.find()` method isn't supported in IE, I've decided that it's best to use the `ban` rule to disallow this.

```json
"ban": {
  "options": [
    {
      "name": ["*", "find"],
      "message": "The 'array.find()' method is not supported by Internet Explorer, so it can't be used in a WebApp project!"}
  ]
}
```

This works, but an unintended side effect is that is now also bans the same method name from calls to jQuery and Underscore!

```ts
$("#foo").find(".bar");
_.find(someArray, (x) => x === "foo");
```

This causes our application to either have a lot of false positives, or have a lot of rule disables.  I know this rule doesn't support type-checking, but is there a way to possibly whitelist jquery/underscore calls?
